games-fps/quakespasm/quakespasm-0.96.0~git.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="An engine for id software's Quake"
+DESCRIPTION="QuakeSpasm is a *Nix friendly Quake Engine based on the SDL port of the  popular  \
+FitzQuake.
+It includes some new features, important fixes, and aims for portability and 64 bit correctness.
+
+Features:
+
+- Supports Windows, Mac OS X, Linux, other unices.
+- Well tested on both 32 bit and 64 bit platforms. Well tested on both little endian and big \
+endian platforms.
+- OpenGL improvements. In-game resolution changing.
+- Widescreen FOV (field of view) support and automatic Hor+ FOV scaling.
+- Improved sound support. Support for OGG, MP3, Opus and WAV external music files.
+- Improved game controller support using SDL2.
+- Countless bug fixes, security fixes and other improvements.
+- Support for playing the 'Quake 2021 re-release' content"
+HOMEPAGE="https://quakespasm.sourceforge.net/"
+COPYRIGHT="
+	1996-2001 Id Software, Inc.
+	2002-2009 John Fitzgibbons and others
+	2010-2014 QuakeSpasm developers
+	"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="70df2b661e9c632d04825b259e63ad58c29c01ac"
+SOURCE_URI="https://github.com/sezero/quakespasm/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="b89ba914d061376945b1fbe2cd0631e162f4f0d835e004eba87a0a9cce107ad4"
+SOURCE_DIR="quakespasm-$srcGitRev"
+ADDITIONAL_FILES="QuakeSpasm.rdef.in"
+PATCHES="quakespasm-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	quakespasm$secondaryArchSuffix = $portVersion
+	cmd:quakespasm$secondaryArchSuffix = $portVersion
```

Comment:
The command is cp'd into $appsDir, didn't this trigger at the least a warning when packaging?
Should be: `app:QuakeSpasm` I think (didn't check yet).